### PR TITLE
Plotly models

### PR DESCRIPTION
**Pull request by hnykda**
Replaces https://github.com/epidemics/covid/pull/201

### CODE REVIEW

**Comment by hnykda** [comment]
@Bachibouzouk I removed the containment_measures for now, as we don't display them yet (can get them back as soon as we agree how)

**Comment by wolverdude** [comment]
This should be ready for merge.

Since this was already a big rewrite, I took the liberty to a little refactoring & lint fixing.

**Comment by hnykda** [comment]
Thanks for the refactoring :+1:

File: src/server/static/js/lines.js [hunk 1]
```diff
@@ -171,387 +60,264 @@ function updateInfectionTotals() {
...
-    infectedPer1000 = data.regions[selected.country].data.infected_per_1000;
-    activeData = infectedPer1000.mitigations[selected.mitigation];
-
-    var countryName = listOfCountries.find(c => c.key == selected.country).value
-    // run the updateChart function with this selected option
-    update();
-    // update the containment measures with the new selected country
-    // update_containment_measures(countryName);
-    // update the name of the country in the text below the graph
-    update_country_in_text(countryName);
```
Comment (hnykda) [inline src/server/static/js/lines.js]: Removing this (or changing some parts) broke the JS fill in the text below the graph. 
image

Comment (hnykda) [reply src/server/static/js/lines.js]: https://github.com/epidemics/covid/issues/217

**Pull request merged**